be/src/storage/sstable/table.cpp

```diff
@@ -250,12 +250,14 @@ Status Table::MultiGet(const ReadOptions& options, const Slice* keys, ForwardIt
         return false;
     };
 
+    size_t i = 0;
     for (auto it = begin; it != end; ++it) {
```

Comment:
for (auto it = begin; it != end; ++it, ++i)

Reply:
done